graphql_api/types/repository/repository.graphql

```diff
@@ -95,19 +96,9 @@ type Repository {
 
   "CoverageAnalytics are fields related to Codecov's Coverage product offering"
   coverageAnalytics: CoverageAnalytics
-  testResultsAggregates: TestResultsAggregates
-  flakeAggregates: FlakeAggregates
```

Comment:
`testResultsAggregates` and `flakeAggregates` are not yet being used by frontend client so moving them to `testAnalytics` as a breaking change